### DISCUSSION TITLE
Relational database entries

### DISCUSSION THREAD

**Discussion by iphelix** [jheddings/notional#29] [Q&A] [answered]
Thank you for writing this awesome integration! 

Could you share some guidance on how to create and edit relational and rollup database entries?

**Comment by jheddings** [comment, marked as answer]
Great question!  It definitely highlights where this library is still in development...  😜 

I created an example that might get you started.  Take a look and let me know if this is headed in the right direction:
[catalog.py](https://github.com/jheddings/notional/blob/main/examples/catalog.py)

I have not had the chance to debug this fully, but I'll keep at it.

**Reply by iphelix** [reply]
That works great! Thank you!
